test/test_mxnet.py

```diff
@@ -221,32 +221,6 @@ def test_horovod_allreduce_type_error(self):
         except (MXNetError, RuntimeError):
             pass
 
-    @unittest.skipUnless(has_gpu, "no gpu detected")
-    def test_horovod_allreduce_cpu_gpu_error(self):
```

Comment:
Can we keep this test, but enable it only if Horovod was built with HORVOD_GPU_ALLREDUCE?  Maybe it's a good time to add functionality to `horovod.common` to expose flags it was built with.

Reply:
I thought about it but there seems no way for me to decide if it needs to run. I can keep it but skip it for both CPU/GPU tests. Later on we can enable it when we have a way to know the flags.

Reply:
Sounds good, let's just disable them, and add a TODO comment to fix it later.

Reply:
Done.